docs/interop.md

```diff
@@ -71,35 +71,39 @@ TInterop interop = objRef.AsInterface<TInterop>();
 ```
 
 ### Custom Marshaling
-When defining a [ComImport](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.comimportattribute?view=netcore-3.1) interop interface, marshaling of WinRT types must be manual (not using the runtime).  
+**Note:** When defining a [ComImport](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.comimportattribute?view=netcore-3.1) interop interface, WinRT parameters and return values must be passed by their ABI types, and marshaling must be done manually (not using the CLR).  For example, reference types like strings and interfaces must be passed as IntPtr.  Blittable value types can be passed directly. Non-blittable value types must have separate ABI and projected definitions, and marshaling between these values must be done manually.

```

Comment:
Do we need our own documentation (list out) on what we consider a Blittable type or does .NET documentation cover it?

Reply:
added link to blittable .NET docs